packages/node_modules/@webex/plugin-memberships/test/integration/spec/memberships.js

```diff
@@ -145,6 +145,52 @@ describe('plugin-memberships', function () {
               validateMembershipEvent(event2, membership, actor);
             }));
       });
+
+      it('creates a space and memberships simultaniously', () => {
+        // "Creating" a space with users generates a unique event
+        // activity object shape via the client that contains
+        // multiple "participants". There isn't a documented
+        // way to perform this action via the API with more than
+        // one user at the time of edit.
+        const created = new Promise((resolve) => {
+          webex.memberships.on('created', (event) => {
+            debug('membership:created event handler for new space test called');
+            resolve(event);
+          });
+        });
+
+        let event;
+
+        return webex.memberships.listen()
+          // Cleanup current room, we need to catch the event when it's created.
+          .then(() => webex.rooms.remove(room))
+          // Create a room to trigger created event.
+          .then(() => webex.rooms.create({title: 'Webex Teams Test Room'}))
+          // Store new room object and wait for event trigger.
+          .then((r) => {
+            room = r;
+
+            return created;
+          })
+          // Store event and get memberships of room.
+          .then((e) => {
+            event = e;
+
+            return webex.memberships.list({roomId: room.id});
+          })
+          // validate data
+          .then((m) => {
```

Comment:
you can deconstruct membership here
`.then(({ items: [membership] }) => {`
^that should work right?

Reply:
Latest commit implements array destructuring as defined. This conflicts with the assertion methods @jpjpjp recommended in a comment. But, a compromise was made in the latest commit.